### PR TITLE
Add Solana DEXs to dex.info

### PR DESCRIPTION
**Pull request by hildobby**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/dex/dex_info.sql [hunk 1]
```diff
@@ -2,7 +2,7 @@
         tags = [ 'static'],
         schema='dex',
         alias = 'info',
-        post_hook='{{ expose_spells(\'["ethereum", "polygon", "bnb", "avalanche_c", "gnosis", "fantom", "optimism", "arbitrum", "celo", "base"]\',
+        post_hook='{{ expose_spells(\'["ethereum", "polygon", "bnb", "avalanche_c", "gnosis", "fantom", "optimism", "arbitrum", "celo", "base", "soalana"]\',
```
Comment (jeff-dude) [inline models/dex/dex_info.sql:5]:
```suggestion
        post_hook='{{ expose_spells(\'["ethereum", "polygon", "bnb", "avalanche_c", "gnosis", "fantom", "optimism", "arbitrum", "celo", "base", "solana"]\',
```

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Comment by jeff-dude** [comment]
plz also fix outstanding merge conflicts

**Review by jeff-dude — APPROVED** [review]
this may not be needed, if the cow PR is merged first?

we'll merge main into this branch after that is merged, so how it looks

**Comment by jeff-dude** [comment]
to be considered after #5071 is merged

**Comment by hildobby** [comment]
> to be considered after #5071 is merged

closing chnges from here are included in https://github.com/duneanalytics/spellbook/pull/5071

**Pull request closed without merging**